pkg/providers/launchtemplate/suite_test.go

```diff
@@ -2245,6 +2245,15 @@ func ExpectLaunchTemplatesCreatedWithUserDataContaining(substrings ...string) {
 	})
 }
 
+func ExpectLaunchTemplatesCreatedWithIPv6PrefixAndPrimaryIPv6() {
```

Comment:
You added this check but then you didn't add the usage of this anywhere -- can we have a test that calls this function so that we know this is working functionally?

Reply:
hmm, sorry. That was missed. Let me add it